dockerfile-buildx-to-build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+for i in $(find . -name Dockerfile.buildx -print); do
+    out=$(echo "$i" | sed 's/x$//')
```

Comment:
Don't we want "${i}" even if it's a one-letter variable?

Reply:
Done.